client/gutenberg/extensions/business-hours/components/day.jsx

```diff
@@ -0,0 +1,210 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+import { Component, Fragment } from '@wordpress/element';
+import { IconButton, TextControl, ToggleControl } from '@wordpress/components';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+
+const defaultOpen = '09:00';
+const defaultClose = '17:00';
+
+class Day extends Component {
+	renderHour = ( hour, hourIndex ) => {
```

Comment:
I think this could be its own component :) - 
I don't think we should refactor it as part of this PR. But I think it would be helpful to do it as well.

Reply:
i agree - i'll do it in a new PR!